### ISSUE TITLE
Issues with latest docker image regarding face-recognition support

### ISSUE THREAD

**Issue by Moep90** [matiasdelellis/facerecognition#509]
See all informations here:
https://github.com/nextcloud/docker/issues/1575

**Comment by Moep90** [comment]
Solved by: https://github.com/Imagick/imagick/issues/331#issuecomment-909938346

**Comment by KoMa1012** [comment]
> Solved by: [Imagick/imagick#331 (comment)](https://github.com/Imagick/imagick/issues/331#issuecomment-909938346)

This solution isn't working since the update of the docker container provided 21st of March! Will now throw:

```
PHP Warning: PHP Startup: Unable to load dynamic library 'pdlib' (tried: /usr/lib/php7/modules/pdlib (Error loading shared library /usr/lib/php7/modules/pdlib: No such file or directory), /usr/lib/php7/modules/pdlib.so (Error relocating /usr/lib/php7/modules/pdlib.so: _ZSt28__throw_bad_array_new_lengthv: symbol not found)) in Unknown on line 0
```
I've been trying a bit, unfortunately couldn't find any way around it, I'm facing the same issue with the PHP8 version (of course with installing the respective php8 versions).

I will keep looking, however ideas or fixes are welcome!

**Comment by ther3zz** [comment]
> > Solved by: [Imagick/imagick#331 (comment)](https://github.com/Imagick/imagick/issues/331#issuecomment-909938346)
> 
> This solution isn't working since the update of the docker container provided 21st of March! Will now throw:
> 
> ```
> PHP Warning: PHP Startup: Unable to load dynamic library 'pdlib' (tried: /usr/lib/php7/modules/pdlib (Error loading shared library /usr/lib/php7/modules/pdlib: No such file or directory), /usr/lib/php7/modules/pdlib.so (Error relocating /usr/lib/php7/modules/pdlib.so: _ZSt28__throw_bad_array_new_lengthv: symbol not found)) in Unknown on line 0
> ```
> 
> I've been trying a bit, unfortunately couldn't find any way around it, I'm facing the same issue with the PHP8 version (of course with installing the respective php8 versions).
> 
> I will keep looking, however ideas or fixes are welcome!

I'm also getting that " _ZSt28__throw_bad_array_new_lengthv: symbol not found" error (have been for a long time now). It was working previously then an update somewhere broke things.

Did you find a solution?
